File: 1.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        for k, v in enumerate(nums):
            b = target - v
            try:
                 i = nums.index(b, k+1)
            except Exception:
                b = 0
            else:
                return [k, i]
        return []

File: 1.go
func twoSum(nums []int, target int) []int {
	if len(nums) < 2 {
		return []int{}
	}
	for i, x := range nums {
		y := target - x
		for j, m := range nums {
			if i != j && m == y {
				return []int{i, j}
			}
		}
	}
	return []int{}
}